### PR TITLE
Filter map

### PR DESCRIPTION
**Pull request by rOmAiin062**

### CODE REVIEW

File: app/src/main/java/science/apolline/view/activity/SplashScreen.kt [hunk 1]
```diff
@@ -138,6 +137,9 @@ class SplashScreen : RootActivity(), AnkoLogger {
                         }
                         BluetoothAdapter.ACTION_DISCOVERY_FINISHED -> {
                             ripple_scan_view.stopRippleAnimation()
+                            //setCurrentItemText()
+                            //selected_textView.text = getString(R.string.splash_bluetooth_scan_finished)
```
Comment (rouvoy) [inline app/src/main/java/science/apolline/view/activity/SplashScreen.kt]: Remove code comments

File: app/src/main/java/science/apolline/view/activity/SplashScreen.kt [hunk 2]
```diff
@@ -138,6 +137,9 @@ class SplashScreen : RootActivity(), AnkoLogger {
...
+
```
Comment (rouvoy) [inline app/src/main/java/science/apolline/view/activity/SplashScreen.kt]: why inserting empty lines ?

File: app/src/main/java/science/apolline/view/activity/SplashScreen.kt [hunk 3]
```diff
@@ -328,60 +333,56 @@ class SplashScreen : RootActivity(), AnkoLogger {
...
+                BluetoothClass.Device.Major.PHONE -> {
+                    onAddClick(circle_layout, nameOrcode, R.drawable.ic_device_bluetooth_phone, isCompatible, isBounded)
+                }
 
-                    BluetoothClass.Device.Major.PHONE -> {
-                        onAddClick(circle_layout, nameOrcode, R.drawable.ic_device_bluetooth_phone, isCompatible, isBounded)
-                    }
+                BluetoothClass.Device.Major.WEARABLE -> {
+                    onAddClick(circle_layout, nameOrcode, R.drawable.ic_device_bluetooth_wear, isCompatible, isBounded)
+                }
```
Comment (rouvoy) [inline app/src/main/java/science/apolline/view/activity/SplashScreen.kt]: It looks like the above piece of code could be refactored

File: app/src/main/java/science/apolline/view/activity/SplashScreen.kt [hunk 4]
```diff
@@ -435,36 +436,30 @@ class SplashScreen : RootActivity(), AnkoLogger {
...
+            } else {
+                newMenu.setBackgroundResource(R.drawable.circle_menu_shape_item)
             }
 
-
+            willBeWhite.mutate().setColorFilter(Color.WHITE, PorterDuff.Mode.SRC_ATOP)
             newMenu.setPadding(20, 20, 20, 20)
             newMenu.setImageDrawable(willBeWhite)
             newMenu.name = name
+            //newMenu.setBackgroundResource(R.color.colorPrimary)
```
Comment (rouvoy) [inline app/src/main/java/science/apolline/view/activity/SplashScreen.kt]: remove if not used

**Comment by rOmAiin062** [comment]
Branch wasn't up to date on develop....

**Pull request closed without merging**